lua/plenary/job.lua

```diff
@@ -319,7 +323,7 @@ function Job:_prepare_pipes()
   end
 
   if not self.stdin then
-    self.stdin = uv.new_pipe(false)
+    self.stdin = (self.interactive == true) and uv.new_pipe(false) or nil
```

Comment:
you can just do `self.stdin = self.interactive and uv.new_pipe(false) or nil` You don't have to check if it's `true`

Reply:
Makes sense, I wasn't sure because I had issues earlier where some variable was set to "1" when working with `vim.fn.has()` I think it was.